frontend/tests/e2e/search/searchSpecUtil.ts

```diff
@@ -15,26 +20,6 @@ export async function fillSearchInputAndSubmit(term: string, page: Page) {
   await page.click(".usa-search > button[type='submit']");
 }
 
-const characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```

Comment:
a bunch of this stuff was either unused or non-search specific. Removed or moved to general utils file